From df770c6284881e97e9a9d64709efd91dcb697f5a Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Wed, 17 Jul 2024 10:57:18 +0200
Subject: [PATCH] FindSmbClient: dont use pkgconfig includedir

This will map to host includes and lead to:
| CMake Error in xbmc/network/httprequesthandler/python/CMakeLists.txt:
|   Imported target "kodi::SmbClient" includes non-existent path
|
|     "/usr/include/samba-4.0"
---
 cmake/modules/FindSmbClient.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/modules/FindSmbClient.cmake b/cmake/modules/FindSmbClient.cmake
index 159dae52f77..8873002a246 100644
--- a/cmake/modules/FindSmbClient.cmake
+++ b/cmake/modules/FindSmbClient.cmake
@@ -38,7 +38,7 @@ if(NOT TARGET ${APP_NAME_LC}::${CMAKE_FIND_PACKAGE_NAME})
       endif()
 
       # pkgconfig sets SMBCLIENT_INCLUDEDIR, map this to our "standard" variable name
-      set(SMBCLIENT_INCLUDE_DIR ${SMBCLIENT_INCLUDEDIR})
+      #set(SMBCLIENT_INCLUDE_DIR ${SMBCLIENT_INCLUDEDIR})
       set(SMBCLIENT_VERSION ${PC_SMBCLIENT_VERSION})
     else()
       find_path(SMBCLIENT_INCLUDE_DIR NAMES libsmbclient.h)
-- 
2.45.1

